src/freenet/clients/fcp/ClientRequest.java

```diff
@@ -27,6 +27,9 @@
  */
 public abstract class ClientRequest implements Serializable {
 
+    /**
+     * ATTENTION: When incrementing this, please skip version 2. Version 2 had already temporarily
+     * been used by a development branch. */
```

Comment:
I'm not concerned about running out of `long`s. I don't think it's reasonable to bother to document compatibility with highly unstable, unreleased code like this. If someone was running unreleased code and want to avoid it they can edit the code themselves - adding this comment increases clutter and the effort required to change the UID. (Is there even a situation where we're willing to change the UID? I thought that broke backwards compatibility?)

If you still disagree I will give up on this because I want to get this over with.